app/src/androidTest/java/com/lucas/medicaltools/service/MedicalToolsRepositoryTest.kt

```diff
@@ -0,0 +1,96 @@
+package com.lucas.medicaltools.service
+
+import retrofit2.Response
+
+import com.lucas.medical_equip.repository.MedicalTool
+import com.lucas.medical_equip.service.APIService
+import com.lucas.medical_equip.service.RetrofitBuilder
+import com.lucas.medicaltools.MainViewState
+import com.lucas.medicaltools.repository.MedicalToolsRepository
+import io.reactivex.Observable
+import io.reactivex.android.plugins.RxAndroidPlugins
+import io.reactivex.plugins.RxJavaPlugins
+import io.reactivex.schedulers.Schedulers
+import org.junit.After
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import org.mockito.ArgumentMatchers
+
+import org.mockito.Mock
+import org.mockito.Mockito
+import org.mockito.MockitoAnnotations
+import org.mockito.junit.MockitoJUnitRunner
+
+@RunWith(JUnit4::class)
+class MedicalToolsRepositoryTest  {
+
+
+    @Mock
+    private lateinit var medicalToolsRepo: MedicalToolsRepository
```

Comment:
Change the repo to not a static based solution, it is an object, thus you won't have mocking a static class error for Mockito.